src/resources/i18n/translations/en.json

```diff
@@ -64,14 +64,14 @@
   "ARIA HINT - change the colors of the user interface": "Change the colours of the user interface",
   "ARIA HINT - change the privacy setting for your clip": "Require a link to view your clip, or make it public for anyone browsing the app.",
   "ARIA HINT - clear the end time for this clip": "Clear the end time for this clip",
-  "ARIA HINT - clear the new episode indicators for all podcasts": "Clear the new episode indicators for all podcasts",
-  "ARIA HINT - clear the new episode indicators for only this podcast": "Clear the new episode indicators for only this podcast",
+  "ARIA HINT - clear the new episode indicators for all podcasts": "Clear the indication of new episodes for all podcasts",
+  "ARIA HINT - clear the new episode indicators for only this podcast": "Clear the indication this episode being new",
   "ARIA HINT - continue to the Make Clip screen": "Continue to \"Make clip\"",
   "ARIA HINT - create a new playlist": "Create a new playlist",
   "ARIA HINT - current playback speed": "Change the playback speed.",
   "ARIA HINT - Current playback time": "Current playback time",
   "ARIA HINT - Currently selected filter": "Selected filter",
-  "ARIA HINT - delete all the episodes you have downloaded for this podcast": "Delete all downloaded episodeS of this podcast",
+  "ARIA HINT - delete all the episodes you have downloaded for this podcast": "Delete all downloaded episodes of this podcast",
   "ARIA HINT - delete this downloaded episode": "Delete this downloaded episode",
   "ARIA HINT - disable the downloaded episode limit for this podcast": "Turn off the episode downloading limit for this podcast",
```

Comment:
```suggestion
  "ARIA HINT - disable the downloaded episode limit for this podcast": "Allow unlimited downloads of this podcast",
```

Reply:
👍

Reply:
Actually, I think I prefer the  "Remove download limit for this podcast" version. Or possibly "Disable download limit for this podcast".